### ISSUE TITLE
LISN_MVP - Swag paths need to connect once completed

### ISSUE THREAD

**Issue by philosofree** [tunapanda/wp-swag#41]
Here is the issue: I have several swagpaths now, but I can find no way to have one swag move onto the next in the path once it is completed.
There are three parts that appear unfinished or illogical as a user:
(I am using "**who am I**" from https://contemplativefire.net/lisn/swag/map/?mode=full
1 as you can see this swag starts with course content![image]
![image](https://cloud.githubusercontent.com/assets/26029570/25528890/c8911e80-2bed-11e7-8cd1-8ed63e78bcbb.png)

However it is not obvious what to do when you have absorbed the content.
I have indicated that the yellow tab should flash or say NEXT or something !
As well, it would be helpful to have a red NEXT button on the bottom right corner of the content

2 When you are doing course content and you do choose step 2...
![image](https://cloud.githubusercontent.com/assets/26029570/25528985/3b039506-2bee-11e7-9173-dfef5e4173d5.png)
Now you have a series of choices to make, at the end of which you press SUBMIT at bottom right.
You then get
![image](https://cloud.githubusercontent.com/assets/26029570/25529040/70247ce6-2bee-11e7-9ba5-d458c8533890.png)
But now there is no guidance, not even a hint as to what to do next.
I propose that a NEXT button (or similar) at the bottom right will action the procedure to take you to the next swag in the track.

3 So when you finish the course content from step 2 above
Optimally, you will now be taken onto the next swag in the swagtrack, which in this case is
![image](https://cloud.githubusercontent.com/assets/26029570/25529158/f3ffcb60-2bee-11e7-8095-220264b1175b.png)
This is a logical next step

4 The problem is that at the top of this step you will see that tunapanda is saying that in order to get the most out of this swag you should have completed the prior swag.
There should be a setting on the swag to ignore this when designing the track, or to be able to set the accomplishment of a prior track automatically on completion of having viewed the page.

In summary, the workflow of a user should be seamless when a swagtrack does not contain tested or evaluated answers, so long as the minimum mandatory fields have in some way been touched by the user (In the example above, selection of any one or more multi-choice should constitute a pass to the next swag)

You can email me at phil@philcheney,com if you have questions about this.

**Comment by philosofree** [comment]
I also would like to be able to automatically set the completion award once the user has done their selection or reading on a particular swagifact.
